source/includes/table-permissions-self-hosted.rst

```diff
@@ -1,50 +1,52 @@
 ..
-   Comment: The nested lists need blank lines before and after each list
-            plus extra indents 
+   Comment: The nested lists need extra indents.  Keep roles in alphabetic
+            order.
 
 .. list-table::
    :header-rows: 1
+   :stub-columns: 1
+   :widths: 20 40 40
 
    * - Sync Type
-     - Target
-     - Required Permissions
-
-   * - default
-     - source cluster
-     -
-
-         - readAnyDatabase
-         - backup
-         - clusterMonitor
-
-   * - default
-     - destination cluster
-     -
-
-         - readWriteAnyDatabase
-         - restore
-         - clusterMonitor
-         - clusterManager
-
-   * - write-blocking or reversing
-     - source cluster
-     -  
-
-         - readWriteAnyDatabase
-         - backup
-         - restore
-         - clusterMonitor
-         - clusterManager
-
-   * - write-blocking or reversing
-     - destination cluster
-     -
-
-         - readWriteAnyDatabase
-         - backup
-         - restore
-         - clusterMonitor
-         - clusterManager
+     - Required Source Permissions
+     - Required Destination Permissions
+
+   * - Default
+     - - :authrole:`backup`
+       - :authrole:`clusterMonitor`
+       - :authrole:`readAnyDatabase`
+
+     - - :authrole:`clusterManager`
+       - :authrole:`clusterMonitor`
+       - :authrole:`readWriteAnyDatabase`
+       - :authrole:`restore`
+
+   * - Write-blocking
+     - - :authrole:`backup`
+        - :authrole:`clusterManager`
+        - :authrole:`clusterMonitor`
+        - :authrole:`readWriteAnyDatabase`
+        - :authrole:`restore`
+
+     - - :authrole:`backup`
+       - :authrole:`clusterManager`
+       - :authrole:`clusterMonitor`
+       - :authrole:`readWriteAnyDatabase`
+       - :authrole:`restore`
+
+   * - Reversing
+     - - :authrole:`backup`
+        - :authrole:`clusterManager`
+        - :authrole:`clusterMonitor`
+        - :authrole:`readWriteAnyDatabase`
+        - :authrole:`restore`
+        - :authrole:`dbRole`
+
+     - - :authrole:`backup`
+       - :authrole:`clusterManager`
+       - :authrole:`clusterMonitor`
+       - :authrole:`readWriteAnyDatabase`
```

Comment:
For reversing the user will need the dbAdmin role on the original destination cluster.

```suggestion
       - :authrole:`dbAdmin`
       - :authrole:`readWriteAnyDatabase`
```